Plugins/Bitbucket/Bitbucket.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
+    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
     <PackageReference Include="RestSharp" Version="106.2.1" />
```

Comment:
Unable to use `$(RestSharpVersion)` due to API breaking changes. This will need to be investigated separately.

Reply:
I can look at that later

Reply:
Thank you!